### ISSUE TITLE
I want to change the convolution kernel shape

### ISSUE THREAD

**Issue by peiyunju** [xwjabc/hed#54]
Hello, may I ask, if my dataset is composed of some large number of straight lines, then is it okay for me to change the convolution kernel to a rectangular convolution kernel.For example, I only care about the gradients on the left and right sides of a line, and not about the gradients up and down, so that I can better identify straight lines.

**Comment by xwjabc** [comment]
@peiyunju  Yeah I think you can give it a try! Feel free to post some results here if you find such kernel pretty useful.
